include/glow/Base/Type.h

```diff
@@ -352,6 +352,34 @@ struct Type final {
     GLOW_UNREACHABLE("Invalid type.");
   }
 
+  /// \return the minimum value of quantized elements.
+  int32_t getQMinValue() const {
```

Comment:
should we just make this generic and return min value for any supported type, getMinValue()?
Could you also add a test to verify that min/max value returned is indeed what you expect?

Current implementation aborts when you call getQMinValue() seems potentially dangerous.

Reply:
The problem is that expressing the limit of different types need a different C type. For example the float limits can't be expressed as an int32_t or int64_t and reverselly, int32_t or int64_t limits can't be expressed accurately with a float. This is why I have decided to limit this functionality to quantized types and to pick int32_t to represent their limit.

Reply:
I changed a bit the code of the PR. In introduced `Type::getQuantizedValueRange()` instead, that gives the float value range represented by quantized type. It's really specific to quantized type because it's not the actual range of elements (that would be integers). I added an `assert(isQuantizedType())` consistently with `getOffset()` and `getRange()` and the code will similarly not abort anymore in case it's compiled in release mode.

Reply:
@rdzhabarov I added some basic tests for quantized types.